src/main/java/com/mg105/entities/items/UpgradeToken.java

```diff
@@ -1,17 +1,31 @@
 package com.mg105.entities.items;
 
+import com.mg105.entities.BattleCharacter;
+import com.mg105.entities.Consumable;
 import com.mg105.entities.Item;
 import com.mg105.utils.ItemConstants;
 
-public class UpgradeToken extends Item {
+public class UpgradeToken extends Item implements Consumable {
 
     /**
      * An item that is used upgrade a characters stat
      *
-     * <p>One upgrade token is equal to a boost a single stat of a single character</p>
+     * <p>One upgrade token boosts every stat of a single character</p>
```

Comment:
Should the upgrade token boost a single stat or all stats on the character? I thought it was going to increase one stat only